src/inputs/TextFieldBase.tsx

```diff
@@ -162,88 +162,86 @@ export function TextFieldBase<X extends Only<TextFieldXss, X>>(props: TextFieldB
           {...tid.label}
         />
       )}
-      {readOnly && (
-        <div
-          css={{
-            // Use input wrapper to get common styles, but then we need to override some
-            ...fieldStyles.inputWrapperReadOnly,
-            ...(multiline ? Css.fdc.aifs.childGap2.$ : Css.truncate.$),
-            ...xss,
-          }}
-          data-readonly="true"
-          {...tid}
-        >
-          {!multiline && inlineLabel && label && !hideLabel && (
-            <InlineLabel labelProps={labelProps} label={label} {...tid.label} />
-          )}
-          {multiline
-            ? (inputProps.value as string | undefined)?.split("\n\n").map((p, i) => (
-                <p key={i} css={Css.my1.$}>
-                  {p.split("\n").map((sentence, j) => (
-                    <span key={j}>
-                      {sentence}
-                      <br />
-                    </span>
-                  ))}
-                </p>
-              ))
-            : inputProps.value}
-        </div>
-      )}
-      {!readOnly &&
-        maybeTooltip({
-          title: tooltip,
-          placement: "top",
-          children: (
-            <div
+      {maybeTooltip({
```

Comment:
No, `maybeTooltip` uses the same props as Tooltip, which requires the `children` prop. See https://github.com/homebound-team/beam/blob/53be1f0a250c8685dfe892837bb4e9120a69dfd9/src/components/Tooltip.tsx#L83
The idea behind maybeTooltip is to conditionally wrap `children` in a Tooltip based on the existence of the `title` prop.

Reply:
Ahh I see, so if we don't have a `title` are we still showing a Tooltip but with the given Children component?

Reply:
No, kind of the opposite. The logic is, if we have a `title`, then wrap `children` in the Tooltip component, otherwise, just return the `children`. 
See the implementation:
```
return props.title ? <Tooltip {...props} /> : props.children;
```
This allows us to remove the logic that used to be in each component where we did something like:
```
// in Button.tsx
if (disabledTooltip || tooltip) {
  return <Tooltip title={disabledTooltip || tooltip}>{button}</Tooltip>
}
return button;
```
Now we can just do:
```
return maybeTooltip({ title: disabledTooltip || tooltip, children: button });
```